gke-deploy/deployer/deployer.go

```diff
@@ -126,28 +116,25 @@ func (d *Deployer) Prepare(ctx context.Context, im name.Reference, appName, appV
 			return fmt.Errorf("failed to check if Namespace %q exists: %v", namespace, err)
 		}
 		if !ok {
-			fmt.Printf("Creating Namespace resource %q\n", namespace)
+			fmt.Printf("Creating Namespace object %q\n", namespace)
```

Comment:
Would it be better to just say "Creating Kubernetes Namespace" or "Creating Namespace"

Reply:
Added a change